doc/cephfs/fs-volumes.rst

```diff
@@ -179,49 +180,49 @@ Fetch the metadata of a subvolume group using::
 
     $ ceph fs subvolumegroup info <vol_name> <group_name>
 
-The output format is json and contains fields as follows.
+The output format is JSON and contains fields as follows:
 
-* atime: access time of subvolume group path in the format "YYYY-MM-DD HH:MM:SS"
-* mtime: modification time of subvolume group path in the format "YYYY-MM-DD HH:MM:SS"
-* ctime: change time of subvolume group path in the format "YYYY-MM-DD HH:MM:SS"
-* uid: uid of subvolume group path
-* gid: gid of subvolume group path
-* mode: mode of subvolume group path
-* mon_addrs: list of monitor addresses
-* bytes_pcent: quota used in percentage if quota is set, else displays "undefined"
-* bytes_quota: quota size in bytes if quota is set, else displays "infinite"
-* bytes_used: current used size of the subvolume group in bytes
-* created_at: time of creation of subvolume group in the format "YYYY-MM-DD HH:MM:SS"
-* data_pool: data pool the subvolume group belongs to
+* ``atime``: access time of subvolume group path in the format "YYYY-MM-DD HH:MM:SS"
+* ``mtime``: modification time of subvolume group path in the format "YYYY-MM-DD HH:MM:SS"
+* ``ctime``: change time of subvolume group path in the format "YYYY-MM-DD HH:MM:SS"
+* ``uid``: uid of subvolume group path
+* ``gid``: gid of subvolume group path
+* ``mode``: mode of subvolume group path
+* ``mon_addrs``: list of monitor addresses
+* ``bytes_pcent``: quota used in percentage if quota is set, else displays "undefined"
+* ``bytes_quota``: quota size in bytes if quota is set, else displays "infinite"
+* ``bytes_used``: current used size of the subvolume group in bytes
+* ``created_at``: time of creation of subvolume group in the format "YYYY-MM-DD HH:MM:SS"
+* ``data_pool``: data pool the subvolume group belongs to
 
 Check the presence of any subvolume group using::
 
     $ ceph fs subvolumegroup exist <vol_name>
 
-The strings returned by the 'exist' command:
+The 'exist' command outputs:
 
 * "subvolumegroup exists": if any subvolumegroup is present
 * "no subvolumegroup exists": if no subvolumegroup is present
 
-.. note:: It checks for the presence of custom groups and not the default one. To validate the emptiness of the volume, subvolumegroup existence check alone is not sufficient. The subvolume existence also needs to be checked as there might be subvolumes in the default group.
+.. note:: It checks for the presence of custom groups and not the default one. To validate the emptiness of the volume, a subvolumegroup existence check alone is not sufficient. Subvolume existence also needs to be checked as there might be subvolumes in the default group.
```

Comment:
s/It/This command/
s/and not the default one/and not the presence of the default one/

Reply:
I puzzled for a while over what this was trying to say.  You're right on both of these counts.  This was the point where I realized I was rabbit-holing.